### DISCUSSION TITLE
Display more than 8 thumbnails in the thumbnail explorer

### DISCUSSION THREAD

**Discussion by YiC200333** [gotson/komga#1540] [Ideas]
### Describe your suggested feature

Is it possible to add an option for the number of thumbnails per page in the settings?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Which thumbnail in which page?

**Comment by YiC200333** [comment]
<img width="1023" alt="捕获" src="https://github.com/gotson/komga/assets/38757441/7bba2f33-31d2-4973-8f6a-c7e4be46880c">
Let every page have more thumbnails, not just eight.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
